__tests__/pages/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
We usually put the tests in the same directory as the tested component. And we name the test `index.spec.js`. But maybe it's not possible with Next?

Reply:
I can try

Reply:
It works :+1: 
Pushed